### DISCUSSION TITLE
Apache SkyWalking Kubernetes not work correctly

### DISCUSSION THREAD

**Discussion by Spartaques** [apache/skywalking#8490] [Q&A]
Hello guys, i attempt to install apache skywalking on my local machine using kubernetes chart. But installation stuck at this point:

![image](https://user-images.githubusercontent.com/29341175/151667192-0cffcd20-610a-4fca-ada0-9b74bfb65fc4.png)

and when i run
mk logs -f elasticsearch-master-0
i see the next:
![image](https://user-images.githubusercontent.com/29341175/151667225-d4c5c5f3-5476-41c1-9946-c30ce5816d26.png)

What i do wrong? Helm please.

**Comment by wu-sheng** [comment]
This log means the ElasticSearch server is not available.

**Comment by Spartaques** [comment]
@wu-sheng i understand this. But it is not available for all users that want to install Skywalking. I ask my college to try install on another computer and he sent me the same error log

**Reply by wu-sheng** [reply]
https://github.com/apache/skywalking-kubernetes/blob/master/test/e2e/e2e.yaml

This is automatical e2e test, including all steps. You could check.

**Comment by kezhenxu94** [comment]
> Hello guys, i attempt to install apache skywalking on my local machine using kubernetes chart. But installation stuck at this point:
> 
> ![image](https://user-images.githubusercontent.com/29341175/151667192-0cffcd20-610a-4fca-ada0-9b74bfb65fc4.png)
> 
> and when i run mk logs -f elasticsearch-master-0 i see the next: ![image](https://user-images.githubusercontent.com/29341175/151667225-d4c5c5f3-5476-41c1-9946-c30ce5816d26.png)
> 
> What i do wrong? Helm please.

You never provide what steps you run, we don't even know what you did, how can we tell you what you did wrong?

> @wu-sheng i understand this. But it is not available for all users that want to install Skywalking. I ask my college to try install on another computer and he sent me the same error log

Being not available for all users might also be the users' misusing it.

You said you install this in your local machine, and you set the ES replicas to 3, and from the screenshot, 2 of them are still pending, that's the problem, ES needs 2 master nodes in this case, but you have only 1 running (2 pending, I guess it's lack of resources as you run this in your local machine), if you've ever tried to make sense why, the logs are very clear below, please try to minimize the installation when you just get started the project

<img width="1044" alt="Screen Shot 2022-01-31 at 10 08 46" src="https://user-images.githubusercontent.com/15965696/151729766-92a03136-8dfa-4d51-8b95-27a371a555fe.png">
